frontend/app/module/users/UsersHomePage.tsx

```diff
@@ -2,21 +2,22 @@ import { Link } from "react-router-dom";
 
 import { NavBar } from "app/component/navbar/NavBar";
 
+import { t } from "@kiesraad/i18n";
 import { PageTitle } from "@kiesraad/ui";
 
 export function UsersHomePage() {
   return (
     <>
       <PageTitle title="Gebruikersbeheer - Abacus" />
       <NavBar>
-        <Link to={"/elections#administrator"}>Verkiezingen</Link>
-        <span className="active">Gebruikers</span>
-        <Link to={"/workstations#administrator"}>Invoerstations</Link>
-        <Link to={"/logs#administratorcoordinator"}>Logs</Link>
+        <Link to={"/elections#administrator"}>{t("election.name_plural")}</Link>
+        <span className="active">{t("users")}</span>
+        <Link to={"/workstations#administrator"}>{t("polling_stations")}</Link>
```

Comment:
```suggestion
        <Link to={"/workstations#administrator"}>{t(""data_entry_stations"")}</Link>
```

Reply:
changed to workstations (workstations.workstations for now)